autoPyTorch/api/base_task.py

```diff
@@ -144,6 +153,26 @@ def __init__(
         search_space_updates: Optional[HyperparameterSearchSpaceUpdates] = None,
         task_type: Optional[str] = None
     ) -> None:
+        """
+
+        Args:
```

Comment:
I think this docstring is WIP :)?

Reply:
oh its not important. We already have the documentation for BaseTask. I'll remove this

Reply:
Looking more into it, please also add `task_type `in the docs for `base_task`